docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -141,6 +143,26 @@ configuration). If the subsystem detects that the `OIDC` mechanism is indeed req
 activate the authentication mechanism automatically. Otherwise, no activation will occur and deployment
 will continue normally.
 
+=== Scope Configuration
+
+OpenID Connect allows the client to use scope values to request access to specific privileges for access tokens. An access token's scopes determine what resources will be available when using the access token to access an OAuth2 protected endpoint. The OpenID Connect https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims[specification] defines a few scope values including:
+
+* `profile` : requests access to the End-User's default profile Claims, such as name, family_name, given_name, middle_name, nickname, preferred_username, profile, picture, website, gender, birthday etc. 
+
+* `email` : requests access to the email and email_verified Claims.
+
+* `address` :  requests access to the address Claim. 
+
+* `phone` : requests access to the phone_number and phone_number_verified Claims.
+
+=== Subsystem Configuration
+It is possible to configure the scope values that would be used in the OIDC authentication request using the `scope` attribute of a `secure-deployment` or a `secure-server` resource in the `elytron-oidc-client` subsystem. The scope values must be added as a space separated list of lowercase ascii values.
```

Comment:
The last sentence here applies to both the subsystem and deployment configuration so maybe we could move this sentence to just before the two sections.

Is "lowercase ascii" values mentioned in the OIDC spec?

Reply:
the Oidc [Spec](https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims:~:text=Multiple%20scope%20values%20MAY%20be%20used%20by%20creating%20a%20space%20delimited%2C%20case%20sensitive%20list%20of%20ASCII%20scope%20values.) mentions that the scope values are case-sensitive.
And yes, I can move it up to the general section.

Reply:
Ok thanks for the pointer. Looks like it mentions "case sensitive" rather than lowercase so we can mention that I think.